projects/storefrontstyles/scss/components/content/banner/_banner.scss

```diff
@@ -1,4 +1,12 @@
 %cx-banner {
+  a {
```

Comment:
I've pushed a change for the focus related rules for banners, to omit specific rules on page templates.